src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/dtos/AdjudicationMigrateDto.kt

```diff
@@ -135,6 +138,37 @@ data class MigratePunishment(
   @Schema(description = "days applied - note we do not support months, should be converted to days if present TBC")
   val days: Int?,
 )
+
+@Schema(description = "hearing and optional result")
+data class MigrateHearing(
+  @Schema(description = "oic hearing id")
+  val oicHearingId: Long,
+  @Schema(description = "oic hearing type")
+  val oicHearingType: OicHearingType,
```

Comment:
we use the same codes ie 
enum class OicHearingType {
  GOV_ADULT,
  GOV_YOI,
  INAD_YOI,
  GOV,
  INAD_ADULT,